code/datums/observation/stat_set.dm

```diff
@@ -13,3 +13,15 @@ GLOBAL_DATUM_INIT(stat_set_event, /decl/observ/stat_set, new)
 /decl/observ/stat_set
 	name = "Stat Set"
 	expected_type = /mob/living
+
+/****************
+* Stat Handling *
+****************/
+//TODO: enable after baymed
+/*
+/mob/living/set_stat(var/new_stat)
+	var/old_stat = stat
+	. = ..()
+	if(stat != old_stat)
+		GLOB.stat_set_event.raise_event(src, old_stat, new_stat)
+*/
```

Comment:
Can't imagine why that was add in that PR.

Reply:
i was checking this part of codebase and noticed that its missing some code, it either need to be completely deleted or mentioned why its missing